openfisca_us/variables/expense/rent.py

```diff
@@ -0,0 +1,10 @@
+from openfisca_us.model_api import *
+
+
+class rent(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Rent"
+    documentation = "Rent"
```

Comment:
I'd favour omitting `documentation` where it's the same.

Reply:
Done here and elsewhere

Reply:
Done